controls/local_policies.rb

```diff
@@ -104,9 +104,7 @@
   ref 'Umsetzungshinweise zum Baustein SYS.1.2.2: Windows Server 2012', url: 'https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Grundschutz/IT-Grundschutz-Modernisierung/UH_Windows_Server_2012.html'
   ref 'Center for Internet Security', url: 'https://www.cisecurity.org/'
   describe security_policy do
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-19' }
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-20' }
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-32-544' }
+    its('SeIncreaseQuotaPrivilege') { should eq ['S-1-5-19', 'S-1-5-20', 'S-1-5-32-544'] }
```

Comment:
Does this still work, if it is not in this exact order?

Reply:
Hi @rndmh3ro ,

I have tested it and is about how Windows manages the list:

By default you have:

```
Administrators
LOCAL SERVICE
NETWORK SERVICE
```

If you remove Administrators group, you get:

```ruby
  ×  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ×  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
     
     expected: ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
          got: ["S-1-5-19", "S-1-5-20"]
     
     (compared using ==)
```

When adding back, Windows returns the list in that exact order always:

```rb
  ✔  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ✔  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
```

If you add an extra user:

```rb
  ×  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ×  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
     
     expected: ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
          got: ["S-1-5-19", "S-1-5-20", "imjoseangel", "S-1-5-32-544"]
     
     (compared using ==)
```

etc

Reply:
Thanks for investigating this!